tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/DomainRule.scala

```diff
@@ -63,7 +63,7 @@ class DomainRule(rewriter: SymbStateRewriter, intRangeCache: IntRangeCache) exte
   }
 
   protected def mkSeqDomain(state: SymbState, seqCell: ArenaCell): SymbState = {
-    val (protoSeq, len, capacity) = proto.unpackSeq(state.arena, seqCell)
+    val (protoSeq @ _, len, capacity) = proto.unpackSeq(state.arena, seqCell)
```

Comment:
`a @ _` is just `a`

Reply:
This silences the warning that `protoSeq` is unused; I figured it makes sense here to see `protoSeq` instead of just binding it to `_`.

Reply:
But why wouldn't you want `_` if `protoSeq` is actually unused?

Reply:
For me, the statement is more readable (i.e., more obvious what `len` and `capacity` refer to and what `unpackSeq` does) if `protoSeq` is named. YMMV.